cypress/support/hostsfunctions.ts

```diff
@@ -85,6 +104,26 @@ function deleteHost(inventoryID: number, host_type: string, hostName: string) {
   cy.contains(/^No results found./);
 }
 
+function deleteHostDetailsView(inventoryID: number, host_type: string, hostName: string) {
+  // function for delete host
+  // can use this for stand alon host and for invntory host
+  // will delete and verify that all was deleted curectlly
+  let url = '';
+  if (host_type === 'inventory_host') {
```

Comment:
We should not be using `cy.visit` and we should not be hard coding url paths in the the tests

Reply:
same

Reply:
done in : https://github.com/ansible/ansible-ui/pull/2491